clients/youtube_producer/js/yt_auth/yt_authDirective.js

```diff
@@ -4,23 +4,10 @@ module.exports = function ($window, $http, $interval, $log) {
 		restrict: 'E',
 		link: function(scope, element, attr){
```

Comment:
Dom Manipulations should not exist in controllers, services or anywhere else but in directives.

from: http://ng-learn.org/2014/01/Dom-Manipulations/